File: sample/sphere_sample/src/main/java/com/sphere/sample/SampleApp.java
package com.sphere.sample;

import android.app.Application;

import com.sphere.analytics.SphereAnalytics;

public class SampleApp extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        // 로그 활성화
        SphereAnalytics.enableLog(true);

        // Sphere Analytics SDK 초기화
        SphereAnalytics.configure(this, "Your Sphere SDK App Key");
    }
}

File: sample/sphere_sample_kt/src/main/java/com/sphere/sample/kt/SampleApp.kt
package com.sphere.sample.kt

import android.app.Application
import com.sphere.analytics.SphereAnalytics

class SampleApp : Application() {

    override fun onCreate() {
        super.onCreate()

        // 로그 활성화
        SphereAnalytics.enableLog(true)

        // Sphere Analytics SDK 초기화
        SphereAnalytics.configure(this, "Your Sphere SDK App Key")
    }
}